mender-auth/Makefile.am

```diff
@@ -0,0 +1,12 @@
+AUTOMAKE_OPTIONS = subdir-objects
+
+ACLOCAL_AMFLAGS = -I m4
+
+AM_CPPFLAGS = -I$(srcdir)/../common/json
```

Comment:
Q: Looking at the includes in the `common/json` files, should this not be:
```
AM_CPPFLAGS = -I$(srcdir)/../
```
? :thinking:

Reply:
Then the same for `update`?